src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/NonScheduledReleaseSummary.test.tsx

```diff
@@ -0,0 +1,418 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import { noop } from 'lodash';
+import { MemoryRouter, Router } from 'react-router-dom';
+import _releaseService, {
+  MyRelease,
+  ReleaseSummary,
+} from '@admin/services/releaseService';
+import userEvent from '@testing-library/user-event';
+import NonScheduledReleaseSummary from '@admin/pages/admin-dashboard/components/NonScheduledReleaseSummary';
+import produce from 'immer';
+import createMemoryHistoryWithMockedPush from '@admin-test/createMemoryHistoryWithMockedPush';
+
+jest.mock('@admin/services/releaseService');
+
+const releaseService = _releaseService as jest.Mocked<typeof _releaseService>;
+
+describe('NonScheduledReleaseSummary', () => {
+  const testRelease: MyRelease = {
+    id: 'rel-3',
+    latestRelease: false,
+    published: '2021-01-01T11:21:17.7585345',
```

Comment:
Would suggest stripping out anything more precise than seconds as it's just noise otherwise

Reply:
Yup no worries, done.  Also tried to replace the instance where I copied this from, but can no longer find it, so maybe it's also been cleared up elsewhere in the meantime.